src/__tests__/JWT-test.ts

```diff
@@ -331,25 +375,12 @@ describe('verifyJWT()', () => {
     )
   })
 
-  it('rejects an nbf in the future', () => {
-    return createJWT({ nbf: NOW + NBF_SKEW + 1 }, { issuer: did, signer }).then(
-      jwt =>
-        verifyJWT(jwt)
-          .catch(error =>
-            expect(error.message).toEqual(
-              'JWT not valid yet (issued in the future): nbf: 1485321434 > now: 1485321133'
-            )
-          )
-          .then(p => expect(p).toBeFalsy())
-    )
-  })
-
   it('accepts a valid exp', () => {
     return createJWT(
-      { exp: NOW - NBF_SKEW + 1 },
-      { issuer: did, signer, expiresIn: 1 }
+      { exp: NOW + 1000 },
```

Comment:
The NBF_SKEW should be the leeway to account for clock skew. That means that a token that expires now should still be considered valid for another NBF_SKEW seconds.

I believe this test should cover something closer to this scenario. `exp: NOW` should work.

Reply:
fixed